src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheckTest.java

```diff
@@ -531,4 +531,21 @@ public void testForEnumConstantDef() throws Exception {
         verifyWithInlineConfigParser(
                 getPath("InputLeftCurlyEnumConstantDef.java"), expected);
     }
+
+    @Test
+    public void commentBeforeLeftCurly() throws Exception {
+        final String[] expected = {
+                "26:34: " + getCheckMessage(MSG_KEY_LINE_PREVIOUS, "{", 34),
```

Comment:
There are some suspicious column numbers in here, please check these in your editor.

Reply:
Yes it was a mistake sorry, now it is done

Done